Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/Dashboard/Ride.hs

```diff
@@ -95,6 +104,11 @@ rideCancel merchantShortId reqRideId Common.CancelRideReq {reasonCode, additiona
           }
   CHandler.dashboardCancelRideHandler CHandler.cancelRideHandle merchant.id rideId dashboardReq
 
+multipleRideCancel :: ShortId DM.Merchant -> Common.MultipleRideCancelReq -> FlowHandler APISuccess
+multipleRideCancel merchantShortId req = do
+  mapM_ (\info -> rideCancel merchantShortId info.rideId Common.CancelRideReq {reasonCode = info.reasonCode, additionalInfo = info.additionalInfo}) req.multiRideCancelReason
```

Comment:
Lets fork this call into a parallel thread.

Reply:
Done